akka-remote/src/main/resources/reference.conf

```diff
@@ -755,8 +755,14 @@ akka {
         #   "<getHostName>"      InetAddress.getLocalHost.getHostName
         #
         hostname = ""
+
+        # Time to wait for Aeron to bind
+        bind-timeout = 1s
```

Comment:
might be too short? could be annoying if it fails spuriously. 3s ?

Reply:
:+1: good idea